src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * Added `MermaidDumper` to dump Workflow graphs in the Mermaid.js flowchart format
```

Comment:
Please follow our new convention for writing changelog entries: https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry

Reply:
If I read that correctly, that would mean present tense `Add  MermaidDumper...` and dropping the patch version, which I will gladly change. It does however feel a bit weird to introduce a style break in the changelog. If that is expected, pretend I never mentioned this (by leaving a thumbs up.)

Reply:
this is expected that we change the convention for writing changelog without rewriting all older changelog content.